### DISCUSSION TITLE
Question about a client created from the API.

### DISCUSSION THREAD

**Discussion by tm6907** [cmu-sei/GHOSTS#281] [Q&A] [answered]
So in the Swagger UI on the API, I created a new client. I guess its not a physical VM like the other Clients I created beforehand so I'm just wondering what timelines can I pass to this new client. 

For example can I copy the block to run Firefox from another VM and get this new client to start generating browsing traffic? If so can I just send that via the /timelines POST

**Comment by sei-dupdyke** [comment, marked as answer]
Since that client has no machine, it can't execute commands. It's meant more creating examples, etc. and was a precursor to NPC which was rolled in from Animator.
